document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/render/RenderDocumentRequestHandler.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.document.generator.core.document.render;
+
+import uk.gov.companieshouse.document.generator.core.document.render.models.RenderDocumentResponse;
+import uk.gov.companieshouse.document.generator.core.document.render.models.RenderDocumentRequest;
+
+import java.io.IOException;
+
+public interface RenderDocumentRequestHandler {
+
+    /**
+     * Call the document render service and convert the data into a document
+     *
+     * @param url
+     * @param request
+     * @return
+     * @throws IOException
+     */
+    RenderDocumentResponse sendDataToDocumentRenderService(String url, RenderDocumentRequest request) throws IOException;
```

Comment:
This interface isn't suitable for other implementations (e.g. if you hook in the SDK) because the SDK won't return an IOException.